docs/GIT/RequestGitHubUserAccess.md

```diff
@@ -11,106 +11,7 @@ BC Gov maintains a set of "organizations" in GitHub where BC Gov digital product
 [bcgov-c](http://github.com/bcgov-c/) - restricted org used by a small number of BC Gov project for storage of source code and artifacts that will ultimately become open source
 [bcdevops](http://github.com/bcdevops/) - organization used for providing access control to DevOps platform, and storage of a set of code and artifacts related to the platform. Not applicable for this document.    
 
-This document describes the process for requesting user access to the `bcgov` and `bcgov-c` organizations.  Access to the DevOps platform (and thereby the `bcdevops`org ) is described elsewhere.  The process for *creating* repositories is also described elsewhere. 
+This document describes the process for requesting user access to the `bcgov` and `bcgov-c` organizations.  Access to the DevOps platform (and thereby the `bcdevops`org ) is described elsewhere. The process for *creating* repositories is also described elsewhere. 
```

Comment:
Is there a link we can use to send people "elsewhere" to read about accessing the platform or creating repositories?